es-vue-base/src/lib-components/EsErrorPage.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <div class="my-5 container">
+        <div class="mb-5 mx-auto col-md-7 text-center">
+            <slot name="errorType" />
+            <slot name="errorMsg" />
+        </div>
+
+        <div class="row">
+            <a
+                :class="linkClass"
+                :style="linkStyle"
+                href="mailto:hello@energysage.com">
+                <div class="circle mb-2">
```

Comment:
Small thing: The color of the circles in figma are the default background color for nuxt pages, so I'm not if there are supposed to be visible circles or not.  It looks fine as-is, but could be worth a quick check with Maria DiPietro if those circles are meant to be visible?